From c6716245271e4a458299209e409a831200771ffc Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Mon, 11 Jan 2021 18:27:18 +0200
Subject: [PATCH] [SW-0] pll: introduce mdelay() function

mdelay() function is useful when delays are getting larger than 1000
useconds.
Specifically, the HBM code will use it.

Change-Id: I6eeaefda1fad507800def1b3186d1a56548338a0
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/common/pll.h b/common/pll.h
index 66a246d..31e494e 100644
--- a/common/pll.h
+++ b/common/pll.h
@@ -57,4 +57,6 @@
 u32 hl_pll_get_freq(u32 pll_base, enum hl_pll_div_out div);
 void udelay(unsigned int usec);
 
+static inline void mdelay(unsigned int msec) { udelay(1000 * msec); }
+
 #endif /* _HL_PLL_H_ */
